custom_components/sector/__init__.py

```diff
@@ -14,8 +14,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: SectorAlarmConfigEntry)
     await coordinator.async_config_entry_first_refresh()
     entry.runtime_data = coordinator
 
+    # Register the listener for configuration updates
     entry.async_on_unload(entry.add_update_listener(async_update_listener))
 
+    # Store the coordinator in hass.data using entry ID for unique identification
+    hass.data.setdefault(entry.entry_id, coordinator)
```

Comment:
We have the coordinator available at `entry.runtime_data` in all places so there should be no need to add it to `hass.data` as well

Reply:
Fixed.